pom.xml

```diff
@@ -82,11 +82,8 @@
         <hbase100.version>${hbase100.core.version}</hbase100.version>
         <jackson1.version>1.9.2</jackson1.version>
         <jackson2.version>2.6.6</jackson2.version>
-        <!-- ES depends on Lucene.  This ES dependency can affect the
-             version used by the janusgraph-lucene module.  When updating
-             the ES version, also consider the version of Lucene, and
-             vice-versa. -->
-        <lucene.version>5.5.2</lucene.version>
+        <lucene.version>6.6.0</lucene.version>
+        <solr.version>${lucene.version}</solr.version>
```

Comment:
@amcp It looks to me like they're both part of one release (see https://github.com/apache/lucene-solr). So I'd actually like to combine these into one property, `lucene-solr.version`. What do you think?